engine/graphics/src/test/wscript

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 
 import waflib.Options
-from waf_dynamo import platform_supports_feature
+from waf_dynamo import platform_supports_feature, platform_glfw_version

```

Comment:
previously missing or unused?

Reply:
It was missing, it only happens when you build with vulkan